ModularTegustation/tegu_items/refinery/crates/corporation.dm

```diff
@@ -161,10 +163,20 @@
 
 	rareloot =	list(
 		/obj/item/gun/ego_gun/shrimp/minigun,
-		/mob/living/simple_animal/hostile/shrimp,
 		/obj/item/grenade/spawnergrenade/shrimp,
+		/obj/item/trait_injector/shrimp_injector,
 	)
 
+	veryrareloot = list(
+		/obj/item/grenade/spawnergrenade/shrimp/super,
+		/obj/item/grenade/spawnergrenade/shrimp/hostile,
+		/obj/item/reagent_containers/pill/shrimptoxin,
+	)
+
+	cosmeticloot = list(
+		/mob/living/simple_animal/hostile/shrimp,
+		/mob/living/carbon/human/species/shrimp,
```

Comment:
This I would actually remove.

Reply:
There we go, did what you told me to so all should be good.